chrome/content/zotero/zoteroPane.js

```diff
@@ -2786,6 +2786,16 @@ var ZoteroPane = new function()
 				return;
 			}
 			
+			//Special treatment for zotero links
+			if (uri.match(/(^zotero\:\/\/)((open-pdf)|(select))/)) {
+				var ioservice = Components.classes["@mozilla.org/network/io-service;1"]
```

Comment:
I like the reduced parentheses  and implemented them in the latest commit - however, the addition of .test(uri) supersedes the special treatment and continues to open the blank window